### PR TITLE
Update volttron dependencies

### PR DESCRIPTION
**Pull request by bonicim**

### CODE REVIEW

**Comment by bonicim** [comment]
Added auto-assign-project workflow.

File: README.md [hunk 1]
```diff
@@ -68,12 +80,12 @@ your Github or Gitlab account. This will be known as our 'remote'.
 
 3. Add the remote (i.e. the new repo URL from your Github or Gitlab account) to your local repository. Run the following command:
 
-```git remote add origin <my github/gitlab URL>```
+```shellgit remote add origin <my github/gitlab URL>```
```
Comment (shwethanidd) [inline README.md:83]: Remove 'shell'

File: README.md [hunk 2]
```diff
@@ -91,19 +103,6 @@ is an example:
...
-
-```
-# form of the command
-# poetry add <path to my local version of volttron-core> --editable
-
-# example; note that the 'volttron-core' is located one directory up from the current directory
-poetry add ../volttron-core/ --editable
-```
-
 # Documentation
```
Comment (shwethanidd) [inline README.md:106]: Remove Documentation section completely.

File: README.md [hunk 3]
```diff
@@ -68,12 +80,12 @@ your Github or Gitlab account. This will be known as our 'remote'.
...
 3. Add the remote (i.e. the new repo URL from your Github or Gitlab account) to your local repository. Run the following command:
 
-```git remote add origin <my github/gitlab URL>```
+```shellgit remote add origin <my github/gitlab URL>```
 
 When you push to your repo, note that the default branch is called 'main'.
 
 
-## Optional Configurations
+# Optional Configurations
```
Comment (shwethanidd) [inline README.md:88]: Remove this section, it's not making sense.

**Review by shwethanidd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bonicim) [reply README.md:83]: Will do.

File: README.md (hunk 3 above)
Comment (bonicim) [reply README.md:88]: Will do. Note that this section came from the template from the copier-agent. We will have to revisit that tool and adjust the template README accordingly.

**Review by shwethanidd — APPROVED** [review]

**Pull request merged**